swift-ticketing/test/swift_ticketing/client.clj

```diff
@@ -1,22 +1,35 @@
 (ns swift-ticketing.client
   (:require
-   [ring.mock.request :as mock]
+   [clj-http.client :as http]
    [clojure.data.json :as json]
-   [swift-ticketing.app :refer [swift-ticketing-app]]
    [swift-ticketing.fixtures :as fixtures]
    [swift-ticketing.factory :as factory]))
 
+(defn- uid-cookie [uid]
```

Comment:
I'm not sure what's going on here. Can you elaborate?
If you're trying to do bi-directional routing, consider libraries like reitit or bidi?

Reply:
I'm creating a cookie here with the value as the `uid` passed to the function. The path `/` denotes that it is accessible to the entire domain. It is merely used for authentication and not for any redirection.